### PR TITLE
fix error message to help with resist fingerprinting

### LINKED ISSUE #3083: Better handle missing mozAddonManager.
I get this page every time I load https://testpilot.firefox.com/ on Nightly.

image

### PR DESCRIPTION
**Pull request by johngruen**
Fixes #3083

### CODE REVIEW

**Comment by johngruen** [comment]
@meandavejustice

File: docs/faq.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ Since Test Pilot is an open source project, you can see all of the cool ways we
 
 ## Why isn't a Test Pilot icon showing up in my toolbar?
 
-Test Pilot can't be installed in [private browsing mode](https://support.mozilla.org/en-US/kb/private-browsing-use-firefox-without-history?redirectlocale=en-US&redirectslug=Private+Browsing),
```
Comment (flodolo) [inline docs/faq.md:15]: DId you really want to remove the trailing space? IIRC it's significant for new lines in markdown.

File: locales/en-US/app.ftl [hunk 2]
```diff
@@ -228,7 +228,7 @@ hidePastExperiments = Hide Past Experiments
 
 ## Text of warnings to the user if various error conditions are detected
 warningGenericTitle = Something is wrong!
-warningGenericDetail = Something has gone wrong with Test Pilot. Please <a>file a bug</a> and mention this error message.
+warningGoToFAQDetail = Something has gone wrong with Test Pilot. Please <a>check the FAQs</a> to learn more.
```
Comment (flodolo) [inline locales/en-US/app.ftl:231]: I'm not sure that "to learn more" is the best choice of words. You should check the FAQ to find possible explanations for the error, or solutions.

Anyway, that's an English problem, l10n wise it's good.

**Review by flodolo — APPROVED** [review]

**Comment by SoftVision-PaulOiegas** [comment]
@johngruen @meandavejustice If I see right, this was reverted. But the strange part is that with `privacy.resistFingerprinting` on `true` I don't get any message on Nightly 61. Also tried this on production with latest Fx 59.0.2 and I only get the Upgrade Firefox button + compatibility message.

Shouldn't there be a modal like the one for local/dev environment setup missing?

**Comment by meandavejustice** [comment]
@SoftVision-PaulOiegas It wasn't reverted, #3487 was closed. The commit is still in master, https://github.com/mozilla/testpilot/commit/db5cf09819e20ac2b2192c3b74e3bf9a0d5523b9. We didn't include it in the latest release because it hasn't been localized

**Comment by SoftVision-PaulOiegas** [comment]
Well that's the problem @meandavejustice. My testing was on dev server (aka master) and I can't see any modal when the pref it's on `true`.

**Pull request merged**